components/enterprise/subscriptions.tf

```diff
@@ -32,6 +32,6 @@ module "tags" {
   for_each    = local.subscriptions
   source      = "git::https://github.com/hmcts/terraform-module-common-tags.git?ref=DTSPO-9743/azure-enterprise"
   environment = try(each.value.environment, (lower(replace([each.key][0], "/.*-([A-Za-z]+).*/", "$1"))))
-  product     = try(each.value.product, (lower(replace([each.value.group][0], "/([A-Za-z])_.*/", "$1"))))
+  product     = try(each.value.product, (tostring(lower(replace([each.value.group][0], "/([A-Za-z])_.*/", "$1"))) == "cft" ? "cft-platform" : tostring(lower(replace([each.value.group][0], "/([A-Za-z])_.*/", "$1"))) == "sds" ? "sds-platform" : tostring(lower(replace([each.value.group][0], "/([A-Za-z])_.*/", "$1")))))
```

Comment:
most of the cft ones have cnp in them although there are some cft ones

Reply:
What grouping in the tfvars would cnp subscriptions come under in prod.tfvars? This is extracting the first section from those groups. Should work for most as is, can we change security_subscriptions to soc_subscriptions to avoid having more logic in here?

Reply:
ah sure yeah that's fine.

hmm that management group is security so I think it would be better to keep it the same.

can't you just use a regex with a capturing group and extract it?

something like:
`"/(security|cft|sds)_.*/"`

and then just add a string replace over top to append -platform ?

and add another entry to the tags module if needed

Reply:
Will give it a look